### PR TITLE
Initial setup

### LINKED ISSUE #2: Setup Repository and FRC Java Project with JUnit.
The standard Overclocked README.md and CONTRIBUTING.md files need to be added to this repo. Then, a standard FRC Java project needs to be created and configuring for use with JUnit. Finally, the Travis.cl badge needs to added.

### PR DESCRIPTION
**Pull request by PaulTerrasi**
@michaelsilver I tried to imitate everything that you did in setting up the dodgeball repo. Everything seems to be working, but obviously you should still look it all over.

Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,62 @@
-# Overclocked-Swerve
+# OverclockedSwerve
+[![Build Status](https://travis-ci.org/246overclocked/OverclockedSwerve.svg)](https://travis-ci.org/246overclocked/OverclockedSwerve)
```
Comment (michaelsilver) [inline README.md]: Should be
```
[![Build Status](https://travis-ci.org/246overclocked/OverclockedSwerve.svg?branch=master)](https://travis-ci.org/246overclocked/OverclockedSwerve)
```

**Comment by michaelsilver** [comment]
Really great job with this!

You should copy over the `.gitignore` from dodgeball, which has all you need -- plus, it will allow you to commit the essential `.classpath` and `.project`. Otherwise, people can't open the project in Eclipse.

Finally, I have a minor correction in the README.

If you do those things, then looks good to me! :rocket:

**Comment by michaelsilver** [comment]
P.S. I didn't fully verify, but I assume you just copied the `lib` folder from dodgeball? If so, you're all good!

**Comment by PaulTerrasi** [comment]
Yes, I did copy it directly. And I fixed the Travis badge.

For `.gitignore`, I removed `.classpath` and `.project`. However, both of the launch configuration files and multiple files in `lib` were supposed to be ignored. To keep the `.gitignore` file consistent with what is actually in the project, I made exceptions for all of those files. If you are OK with that, then I will merge.

**Comment by michaelsilver** [comment]
Great thanks for the fixes! Looking it over again, the `.gitignore` makes sense (I made one small comment), but you still haven't added the `.project` and `.classpath` files to the repo, so I still can't open it in Eclipse.

**Comment by michaelsilver** [comment]
Once you add the `.project` and `.classpath` files to the repo, and address my minor comment about the .gitignore, you should be all set to merge.

**Comment by PaulTerrasi** [comment]
I removed that thing from .gitignore because it also wasn't in Dodgeball's
.gitignore file. I am just going to leave it out, I'm pretty certain it's
not important.

I will add those files later tonight.

**Pull request merged**